lib/msf/util/document_generator/document_normalizer.rb

```diff
@@ -185,6 +185,25 @@ def normalize_targets(targets)
           targets.collect { |c| "* #{c.name}" } * "\n"
         end
 
+        # Returns the markdown format for module AttackerKB references.
+        #
+        # @param refs [Array] AttackerKB references , with the CVE as the key and the link as the value.
+        # @return [String]
+        def normalize_attackerkb_references(refs)
+          normalized = ''
+
+          # Grabs module CVE's if available and adds each AttackerKB link to the hash
+          unless refs.empty?
```

Comment:
This is redundant - if refs was empty, it just wouldn't loop over anything 👀 

```suggestion
```

Reply:
Ah sorry, I think that was left over from my implementation from module hacking. I'll get that removed.